Packs/Cybereason/Integrations/Cybereason/Cybereason.py

```diff
@@ -308,7 +311,7 @@ def query_processes_command(client: Client, args: dict):
 
     return CommandResults(
         readable_output=tableToMarkdown('Cybereason Processes', outputs, headers=PROCESS_HEADERS),
-        outputs_prefix='Cybereason.Process',
+        outputs_prefix='Process',
```

Comment:
I strongly advice against this change. Think about a case where a client is using both you product and another one that can search for processes. Then the client needs to be able to know what came from your integration and what came for another one.

Reply:
Sure Jawine.
That makes complete sense. We will add the Cybereason Prefix again.
Thanks for pointing it out.

Reply:
Fixed now